packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -156,6 +156,14 @@ export class CdkErrorMapper {
       errorName: 'CDKVersionMismatchError',
       classification: 'ERROR',
     },
+    {
+      errorRegex: /Command cdk not found/,
+      humanReadableErrorMessage: 'Unable to detect cdk installation',
+      resolutionMessage:
+        "Install the dependencies in your project by running 'yarn install' or 'npm install'",
```

Comment:
```suggestion
        "Make sure that dependencies in your project are installed with your package manager.",
```

or `"Make sure that dependencies in your project are installed with your package manager. For example, by running 'yarn install' or 'npm install'",`

We also support pnpm. The message should be formed in such a way that customer is encouraged to try their desired package manager. Otherwise some will just run these commands blindly.

Reply:
updated